examples/middlewares/marketcatalogue.py

```diff
@@ -0,0 +1,17 @@
+import json
+import os
+
+from betfairlightweight.resources import MarketCatalogue
+from flumine.markets.middleware import Middleware
+
+MARKET_CATALOGUE_PATH = ""
+
```

Comment:
Thanks for this, could you create a small comment on what it is / how to use it?